typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/decisionItemDecoratorStrategy.ts

```diff
@@ -40,19 +39,21 @@ import { formatPosition } from "../helpers/formatPosition";
    <position y="40.0" x="380.0"/>
  </workflow-item>
  */
-export default class DecisionItemDecoratorStrategy implements CanvasItemDecoratorStrategy {
-
-	constructor(private readonly sourceFilePrinter: SourceFilePrinter = new WrapperSourceFilePrinter()) { }
+export default class DecisionItemDecoratorStrategy extends BaseItemDecoratorStrategy {
```

Comment:
I disagree to extending this logic like this, we don't gain anything but complexity..